src/ActionProviders/AuthActions.js

```diff
@@ -0,0 +1,59 @@
+import { UserContext } from '../contextCreator'
+import { useReducer, useContext } from 'react'
+import { userReducers } from '../reducers/authReducer'
+
+const UserProvider = props => {
+	const [
+		{ userInfo, error: userAuthError, laoding: authLoading },
+		dispatch,
+	] = useReducer(userReducers, {
+		userInfo: {},
+	})
+
+	const setAuthLoading = () => {
+		dispatch({
```

Comment:
suggestion: give specific name for dispatch as well. eg: `authDispatch`

Reply:
Actually all my dispatchers which are reused again are wrapped inside functions which have specific names for the action they are performing. Please let me know if I am missing something. Will work on cards

Reply:
Okay, got it. Works that way 👍🏻